packedForest/src/fp.cpp

```diff
@@ -98,7 +106,7 @@ int main(int argc, char* argv[]) {
 
 
 		forest.setParameter("numTreesInForest", 10);
-		forest.setParameter("minParent", 1);
+		forest.setParameter("minParent", 100);
```

Comment:
is this what we think is a good default for urf/urerf or was this just for testing?